docs/source/app_developers_guide/go_sdk_import.rst

```diff
@@ -0,0 +1,59 @@
+********************
+Importing the Go SDK
+********************
+
+.. note::
+   The Sawtooth Go SDK assumes that you have the latest version of Go,
+   more information on installing and configuring can be found at `go
+   <https://github.com/golang/go>`_.
+
+Once you've got a working version of Sawtooth, there are a few additional
+steps you'll need to take to get started developing for Sawtooth in Go.
+
+1. Get Go SDK located at `Sawtooth Go SDK repository
+   <https://github.com/hyperledger/sawtooth-sdk-go>`_.
+
+.. code-block:: console
+    :caption: Run following command for downloading Go SDK
+
+    $ go get github.com/hyperledger/sawtooth-sdk-go
+
+2. Import the SDK into your Go files. You need to specify packages from SDK
```

Comment:
> You need to specify packages from SDK which are needed.

This sentence is not well formed.  Does it mean "You need to specify which packages from the SDK are needed."? If so, I suggest adding "... as in this example:" at the end of the sentence.